mono/mini/interp/interp.c

```diff
@@ -2579,9 +2579,9 @@ ves_exec_method_with_context (MonoInvocation *frame, ThreadContext *context, uns
 			ftndesc.addr = rmethod->jit_addr;
 			ftndesc.arg = NULL;
 
+			gpointer *args = g_malloc (32);
+			guint8 *res_buf = g_malloc (sizeof (gpointer) );
```

Comment:
When using mixed mode, the JIT exception handler might throw away these interpreter frames, so there is nobody to free these.